src/components/UploadingJobsContextProvider/UploadingJobsContextProvider.js

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable react/no-unused-state */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  withStripes,
+  stripesShape,
+} from '@folio/stripes/core';
+
+import {
+  createUrl,
+  createOkapiHeaders,
+} from '../../utils';
+import { UploadingJobsContext } from '.';
+
+class UploadingJobsContextProvider extends Component {
+  static propTypes = {
+    stripes: stripesShape.isRequired,
+    children: PropTypes.oneOfType([
+      PropTypes.arrayOf(PropTypes.node),
+      PropTypes.node,
+    ]).isRequired,
+  };
+
+  constructor(props) {
```

Comment:
Can we get rid of `constructor`?

Reply:
Will not work in the state will be defined outside the constructor